docker/deploy/Dockerfile

```diff
@@ -0,0 +1,70 @@
+ARG BUILD_ENV
+
+FROM $BUILD_ENV as builder
+
+# Add deployment tooling
+RUN wget -q https://github.com/haampie/bundler/releases/download/v0.1.3/bundler_x86_64.tar.gz && \
+    tar -xzf bundler_x86_64.tar.gz && \
+    rm bundler_x86_64.tar.gz
+
+# Build COSMA
+COPY . /COSMA
+
+RUN mkdir /COSMA/build && cd /COSMA/build && \
+    CC=mpicc CXX=mpicxx FC=mpif90 FCCPP=cpp cmake .. \
```

Comment:
I think `FC=mpif90 FCCPP=cpp` are not necessary since we don't use fortran compilers.